docs/ref/settings.txt

```diff
@@ -1899,8 +1899,15 @@ The list is a list of two-tuples in the format
 This specifies which languages are available for language selection. See
 :doc:`/topics/i18n/index`.
 
-Generally, the default value should suffice. Only set this setting if you want
-to restrict language selection to a subset of the Django-provided languages.
+Generally, the default value should suffice. You can set this setting if you
+want to restrict language selection to a subset of the Django-provided
+languages. You can also extend this setting to add languages that are
+not currently supported by Django. For instance you can do::
```

Comment:
Maybe a one sentence form? Also we use "For example" rather than "For instance"

```
Generally, the default value should suffice. You can set this setting if you
want to restrict language selection to a subset of the Django-provided
languages, or to add others not currently supported by Django. For example::
```

Reply:
Agreed, the same should be noted in [How Django discovers language preference](https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#how-django-discovers-language-preference).